From c216f6546d636b18d6d0d70c06ee503226a640e4 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:37:17 -0400
Subject: [alsa] hda: Remove obsoleted SFX definitions

Message-id: <140972823573-365-patchset-jkysela@redhat.com>
Patchwork-id: 91061
O-Subject: [RHEL 7.1 PATCH 364/442] ALSA: hda - Remove obsoleted SFX definitions
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

It's no longer referred by anyone after standardizing with dev_*()
macros.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: c6bf1d8e8c08d37d42a3a96606470fc012cc30e7)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/hda_auto_parser.c b/sound/pci/hda/hda_auto_parser.c
index dabe419..a979019 100644
--- a/sound/pci/hda/hda_auto_parser.c
+++ b/sound/pci/hda/hda_auto_parser.c
@@ -17,8 +17,6 @@
 #include "hda_local.h"
 #include "hda_auto_parser.h"
 
-#define SFX "hda_codec: "
-
 /*
  * Helper for automatic pin configuration
  */
diff --git a/sound/pci/hda/hda_priv.h b/sound/pci/hda/hda_priv.h
index 0688aaf..da2d3a4 100644
--- a/sound/pci/hda/hda_priv.h
+++ b/sound/pci/hda/hda_priv.h
@@ -358,12 +358,6 @@ struct azx {
 #endif
 };
 
-#ifdef CONFIG_SND_VERBOSE_PRINTK
-#define SFX /* nop */
-#else
-#define SFX "hda-intel "
-#endif
-
 #ifdef CONFIG_X86
 #define azx_snoop(chip)  ((chip)->snoop)
 #else
-- 
1.7.1